schemas/oic.wk.mnt-schema.json

```diff
@@ -17,6 +17,13 @@
         "rb": {
           "type": "boolean",
           "description": "Reboot Action"
+        }        ,
+        "err": {
+          "type": "integer",
```

Comment:
As mentioned in the CR, now that we've added one more Property to this Resource, I don't think this makes sense any longer:

>        "anyOf": [
>          {"required": ["fr"]},
>          {"required": ["fr"]},
>          {"required": ["rb"]}
>          {"required": ["rb"]}
>        ],

This will prevent a Device from only implementing the "err" Property which is a possible use case.  This only made sense when there were only two Properties in this schema.